composer.json

```diff
@@ -56,7 +56,5 @@
     "branch-alias": {
       "dev-feature/contao4": "4.0.x-dev"
     }
-  },
-  "prefer-stable": true,
-  "minimum-stability": "dev"
```

Comment:
I removed it because it pulled ALL dependencies as `master` which broke everything and made installing a big PITA.

Installing is no problem when you use (i.e.):
```
       	"contao-community-alliance/dc-general":	"~2.1@dev",
        "contao-community-alliance/dependency-container": "~2.0@dev",
        "contao-community-alliance/event-dispatcher": "~2.0@dev",
       	"contao-community-alliance/events-contao-bindings": "~4.0@dev",
       	"contao-community-alliance/translator":	"~2.1@dev",
       	"contao-community-alliance/url-builder": "~1.3@dev",
        "contaoblackforest/contao-core-dcg": "^1.0@dev",
```